iOS Example Frame/iOS Example Frame/CustomStyle1.swift

```diff
@@ -25,6 +25,29 @@ private enum Constants {
 // MARK: - Payment Form
 // **********************
 
+struct PayButtonFormStyleCustom1: ElementButtonStyle {
+  var image: UIImage?
+  var textAlignment: NSTextAlignment = .center
+  var text: String = "Pay 100$"
+  var font: UIFont = UIFont.systemFont(ofSize: 15)
+  var disabledTextColor: UIColor = Constants.secondaryFontColor
+  var disabledTintColor: UIColor = Constants.secondaryFontColor
+  var activeTintColor: UIColor = Constants.mainFontColor
+  var backgroundColor: UIColor = Constants.mainFontColor
+  var textColor: UIColor = .white
```

Comment:
Some background colour here is failing as the text is invisible when button is disabled.

Reply:
updated to `var disabledTextColor: UIColor = .white`